rollup.config.js

```diff
@@ -141,11 +144,40 @@ const umd = [
   },
 ];
 
-let config = {};
-console.log(process.env.BUILD_ENV);
+const isDevServer = process.env.SERVE === 'true';
+const isLiveReload = process.env.RELOAD === 'true';
+const demoBundleConfig = {
+  input: './docs/js/index.js',
+  output: {
+    preserveModules: false,
+    file: 'docs/demo-bundle.js',
+    format: 'iife', // immediately-invoked function expression — suitable for <script> tags
+    sourcemap: true,
+    banner,
+  },
+  plugins: [
+    resolve(),
+    commonjs(),
+    injectProcessEnv({
+      BUILD_ENV: process.env.BUILD_ENV,
+      IS_CYPRESS_ENV: process.env.IS_CYPRESS_ENV,
+    }),
+    babel({ babelHelpers: 'bundled' }),
+    isDevServer &&
+      serve({
+        port: 9966,
+        contentBase: 'docs',
+      }), // index.html should be in root of project
+    isLiveReload && livereload({ port: 9966, watch: 'docs' }), //
```

Comment:
There's still this empty comment here on line 171. Could you remove it? :)

Reply:
sure